content/_index.md

```diff
@@ -0,0 +1,23 @@
+# Linux as Firmware
+**Reinventing the wheel by implementing drivers for firmware again and again.**</br>
+**Not with LinuxBoot™ !**
+
+![](images/linuxboot_info.png)
```

Comment:
This image is good, though, I like it.

Reply:
* let me finish that sentence: but remove the u-root initramfs part. That's not part of LinuxBoot. NERF = LinuxBoot + u-root initramfs.